src/parser/transforms.js

```diff
@@ -295,7 +315,7 @@ module.exports.preprocessGlimmerTemplates = function preprocessGlimmerTemplates(
         // for blocks {{x as |b|}} the block range does not contain the block params...
```

Comment:
is this part still needed? the blockRange? it come from one of the other fix PRs, but it causes an error on this branch

Reply:
hmm, it should be required, yes